crates/nu-command/src/formats/to/nuon.rs

```diff
@@ -42,14 +42,20 @@ impl Command for ToNuon {
         call: &Call,
         input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
-        let raw = call.has_flag(engine_state, stack, "raw")?;
-        let tabs: Option<usize> = call.get_flag(engine_state, stack, "tabs")?;
-        let indent: Option<usize> = call.get_flag(engine_state, stack, "indent")?;
+        let style = if call.has_flag(engine_state, stack, "raw")? {
+            nuon::ToStyle::Raw
+        } else if let Some(t) = call.get_flag(engine_state, stack, "tabs")? {
+            nuon::ToStyle::Tabs(t)
+        } else if let Some(i) = call.get_flag(engine_state, stack, "indent")? {
+            nuon::ToStyle::Spaces(i)
+        } else {
+            nuon::ToStyle::Raw
```

Comment:
I am not sure when the default and `--raw` became the same?

Reply:
i was like this before i created the `nuon` crate :wink: 

55edef5dd is the commit of the crate creation and fac2f43aa49377488d3dd4247f24534772ebdc8c is the commit before https://github.com/nushell/nushell/blob/fac2f43aa49377488d3dd4247f24534772ebdc8c/crates/nu-command/src/formats/to/nuon.rs#L58-L66

Reply:
The origin of `--raw` being equal to the default is in #8366 and there in turn in an attempt to mirror `to json`.

Reply:
ahah yeah that was me lol